.github/workflows/build.yml

```diff
@@ -248,14 +248,15 @@ jobs:
       - name: Install rust toolchain
         uses: dtolnay/rust-toolchain@nightly
 
-      - name: Update to direct-minimal-versions
-        # This has no effect if no `Cargo.lock` exists yet.
-        run: cargo update -Z direct-minimal-versions
+      - name: Install cargo-minimal-versions
+        uses: taiki-e/install-action@cargo-minimal-versions
 
-      - name: cargo test (debug; all features)
-        run: cargo test --locked --all-features
-        env:
-          RUST_BACKTRACE: 1
+      - name: Install cargo-hack
```

Comment:
Do we need both `cargo-hack` and `cargo-minimal-versions`? Why?

Reply:
`cargo-minimal-versions` shells out to `cargo-hack` -- see https://github.com/taiki-e/cargo-minimal-versions#prerequisites

Reply:
Okay, might want to add a comment in the workflow here?

Reply:
will do